README.md

```diff
@@ -1,2 +1,66 @@
-# ticket-master
-Ticketmaster Entertainment, Inc. is an American ticket sales and distribution company based in Beverly Hills, California with operations in many countries around the world. In 2010, it merged with Live Nation under the name Live Nation Entertainment. Ticketmaster charges a fee on tickets purchased and resold on the platform.
+# MyTicketO - Ticket App
+
+> Ticketmaster Entertainment, Inc. is an American ticket sales and distribution company based in Beverly Hills, California with operations in many countries around the world. In 2010, it merged with Live Nation under the name Live Nation Entertainment. Ticketmaster charges a fee on tickets purchased and resold on the platform.
+
+## Built With
+
+- Major languages: **JavaScript**
+- Library: **React.js**, **React-Redux**, **@reduxjs/toolkit**, **redux-logger**
+- Technologies used: **CSS3, JSX**
+- Testing Tool: **Jest**, **React-testing Library**
+
+## Live Demo (if available)
+
+[click here](https://myticketo.netlify.app)
+
+## Video Presentation
+
+[Click Here](https://www.loom.com/share/c67c334458a94215b27b75e3ed889fb3)
+
+## Getting Started
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+> In order to work with this project. You need to have basic understanding of *React.js*.
+
+### Setup
+
+- git clone `https://github.com/skyv26/ticket-master.git`
+
+### Install
+
+**In order to start the project properly please install the npm dependicies by using below command**
+
+- npm install
+
+### Usage
+
+**In order to run your project use below command**
+
+- npm start
+
+## Authors
+
+👤 **Aakash Verma**
+
+- GitHub: [@skyv26](https://github.com/skyv26)
+- Twitter: [@vrma_aakash](https://twitter.com/vrma_aakash)
+- LinkedIn: [dev.aakashv](https://linkedin.com/in/skyv2022)
+
```

Comment:
The README should contain an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

Reply:
@zainsadaqat I added thanks for pointing out issue.